nrfs/include/services/nrfs_swext.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NRFS_SWEXT_H
+#define NRFS_SWEXT_H
+
+#include <internal/services/nrfs_swext.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief SWEXT control service response notification types. */
+typedef enum __NRFS_PACKED {
+	NRFS_SWEXT_REQ_RESPONSE,	/** Request response. */
+	NRFS_SWEXT_REQ_REJECTED,	/** Request rejected. */
+} nrfs_swext_evt_type_t;
+
+/** @brief SWEXT control service response notification types. */
+typedef enum __NRFS_PACKED {
+	NRFS_SWEXT_OUTPUT_ENABLED,	/** SWEXT powered up correctly. */
+	NRFS_SWEXT_OUTPUT_DISABLED,	/** SWEXT powered down correctly. */
+	NRFS_SWEXT_OVERCURRENT,		/** Overcurrent detected. SWEXT is powered down */
+} nrfs_swext_evt_status_t;
+
+/** @brief SWEXT control service response data structure. */
+typedef struct {
+	nrfs_swext_evt_type_t type;
+	nrfs_swext_evt_status_t status;
+} nrfs_swext_evt_t;
+
+/** @brief SWEXT control service event handler type. */
+typedef void (*nrfs_swext_evt_handler_t)(nrfs_swext_evt_t const *p_evt,
```

Comment:
nit - could be single-lined: `typedef void (*nrfs_swext_evt_handler_t)(nrfs_swext_evt_t const *p_evt, void *context);`